adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -93,13 +95,29 @@ - (void)initWithAppId:(nonnull NSString *)appId
   }
 
   _isInitializing = YES;
-  _isBannerPresenting = NO;
 
   // Disable refresh functionality for all banners
   [[VungleSDK sharedSDK] disableBannerRefresh];
 
+  // Set init options for priority placement
+  NSMutableDictionary *initOptions = [NSMutableDictionary dictionary];
+  if (delegate) {
+    NSString *priorityPlacementID = delegate.desiredPlacement;
+    [initOptions setObject:priorityPlacementID forKey:VungleSDKInitOptionKeyPriorityPlacementID];
+    _prioritizedPlacementID = [priorityPlacementID copy];
+
+    NSInteger priorityPlacementAdSize = 1;
```

Comment:
What is 1? Is that one of your ad size constants? If so, use the constant directly here for code clarity.

Reply:
@ericleich  Hi Eric, yes 1 is for `VungleAdSizeUnknown` enum. I will use `VungleAdSizeUnknown` directly here. Thanks for the guidance. :)

Reply:
@ericleich Updated, thanks a lot. ^_^